xtokens/src/lib.rs

```diff
@@ -571,34 +570,121 @@ pub mod module {
 				assets.len() <= T::MaxAssetsForTransfer::get(),
 				Error::<T>::TooManyAssetsBeingSent
 			);
+			let origin_location = T::AccountIdToMultiLocation::convert(who.clone());
+
+			let fee_to_dest = reset_fee(&fee, dest_weight as u128);
```

Comment:
`dest_weight` is weight not the amount?

Reply:
the `dest_weight` should be convert to fee amount, if using `WeightToFee` trait, it only works on one parachain. so here add another config to xTokens module: `MinXcmFee` which can config many parachain(kind like of `ExistentialDeposits`). parachain don't want to this feature can simply config as `()`.